src/apps/relay/mainrelay.c

```diff
@@ -1619,7 +1619,9 @@ static void read_config_file(int argc, char **argv, int pass)
 				if (!s[0])
 					continue;
 				size_t slen = strlen(s);
-				while (slen && ((s[slen - 1] == 10) || (s[slen - 1] == 13)))
+
+				// strip white-spaces from config file lines end, use arg="value " syntax if you need space at the end
+				while (slen && ((s[slen - 1] == 10) || (s[slen - 1] == 13) || (s[slen - 1] == ' ')))
```

Comment:
I'd suggest using isspace() from stdio.h here, e.g.
```
// strip white-spaces from config file lines end, use arg="value " syntax if you need space at the end
while (slen && isspace(s[slen - 1]))
	s[--slen] = 0;
```
Also, the comment about the arg="value " syntax is something that could be part of the documentation, not just a code comment.

Reply:
Looks good to. This would also include tab. 
I tried to make mimic the original code but "== 32" looked a bit over complicated to me.

I would add to the documentation that values can be surrounded by " " if it's not already there. My example is an edge case that confuses the normal user more than it explains IMHO